env/src/lib.rs

```diff
@@ -283,6 +280,21 @@ impl LogBuilder {
 }
 
 impl Logger {
+    pub fn new() -> Logger {
+        let mut builder = LogBuilder::new();
+
+        if let Ok(s) = env::var("RUST_LOG") {
+            builder.parse(&s);
+        }
+
+        builder.build()
+    }
+
+    pub fn filter(&self) -> LogLevelFilter {
```

Comment:
Not sure if this should be called `filter` or something like `max_level` to match up with the thing passed into `init`?